### ISSUE TITLE
Unable to complete a search using the enter key on Active Consultations page

### ISSUE THREAD

**Issue by rebeccawilliamss** [archesproject/arches-her#454]
#### Describe the issue
Cannot use the keyboard to complete a search, you have to click on the search button. This is an accessibility issue for users who only use the keyboard to navigate a site.

#### Steps to reproduce
1) Open the Active Consultations page http://qa.archesproject.org/consultations/plugins/active-consultations
2) Type in a search term
3) Press the enter key - the enter key does not work to carry out the search

**Comment by StucakeGLHER** [comment]
@rebeccawilliamss @aj-he  You need to press the TAB button first, then enter.  It moves the active feature to the Search button.  Perhaps some instruction is needed here, or somewhere for users who can only use a keyboard?

**Comment by rlwilliamss** [comment]
@StucakeGLHER Thank you, yes maybe we could just add some guidance for users who will be using a keyboard. I'll close this as it does work by tabbing to the Search button.
